src/components/pages/CaseOverview/CaseOverview.js

```diff
@@ -72,10 +72,7 @@ const CaseOverview = props => {
                 {caseData.initial_or_appellate ? 'True' : 'False'}
               </p>
               <p>Case Origin: {caseData.case_origin}</p>
-              <p>
-                Filed 1 Year:{' '}
-                {caseData.filed_within_one_year ? 'True' : 'False'}
-              </p>
+              <p>Filed 1 Year: {caseData.filed_within_one_year}</p>
```

Comment:
The paragraph text on line 75 should probably be changed to "Filed Within One Year" for consistency, or at least changed to "Filed Within 1 Year".